### PR TITLE
Change tf_upgrade_v2 to be compatible with tf1 and tf2 for tf.nn.dropout

### PR DESCRIPTION
**Pull request by ytfksw**
According to [the recommended upgrade process](https://www.tensorflow.org/guide/upgrade#recommended_upgrade_process), even after converting,
We can run and pass the test with tensorflow 1.14 as well.

I found that If we run tf_upgrade_v2 on tf.nn.dropout, tf1 and tf2 are incompatible.
In the case of `tf.nn.dropout(x, keep_prob)`, if you don't specify it
with a keyword, such as, tf_upgrade_v2 converts to `tf.nn.dropout(x, 1-keep_prob)`.

In tf2, it is compatible with tf1 since the second argument is rate and rate=1-keep_prob.
In 1.14, on the other hand, the result is not the same because the second argument is keep_probe.

For compatibility, it should be `tf.nn.dropout(x, rate=1-keep_prob)`.

### CODE REVIEW

**Comment by ytfksw** [comment]
@googlebot I signed it!

**Review by mihaimaruseac — APPROVED** [review]
